### PR TITLE
[Tools] Add script for changing config settings

### PR DESCRIPTION
**Pull request by driusan**
This adds a new script to change the config settings
in the database without SQL access. This may be is useful
when the Config module page is unavailable due to bad
paths or otherwise when providing user support, where
it can be easier to just tell the user what to type
copy/paste in a terminal than describe where to find
a setting on the frontend.

#### Testing instructions (if applicable)

1.

#### Links to related tickets (GitHub, Redmine, ...)

*

### CODE REVIEW

File: tools/setconfig.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+$id= $DB->pselectOne(
+    "SELECT ID FROM ConfigSettings WHERE Name=:config",
+    array('config' => $setting)
+);
+if (!is_numeric($id) || $id <= 0) {
+    die("Invalid config name");
+}
+
+// Determine whether to insert or update
```
Comment (ridz1208) [inline tools/setconfig.php]: you also have to check if its a config of type "multiple"

**Comment by driusan** [comment]
@ridz1208 it's not clear what the behaviour should be for "multiple" settings (add? update? delete?) so I just made it die with an error message.. as far as I know none of the config settings that make the frontend inaccessible are multiple settings.

File: tools/setconfig.php (hunk 1 above)
Comment (johnsaigle) [reply tools/setconfig.php]: I think this is addressed now?

File: tools/setconfig.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/php
+<?php
```
Comment (johnsaigle) [inline tools/setconfig.php]:
```suggestion
<?php declare(strict_types=1);
```

**Comment by johnsaigle** [comment]
Also a small thing but the script gives this error when an invalid config setting is supplied:

php tools/setconfig.php notReal true

> PHP Warning:  count(): Parameter must be an array or an object that implements Countable in /var/www/loris/tools/setconfig.php on line 44

I think the DB function returns `null` not `array()` when no value is found so `count()` throws a warning.

**Comment by driusan** [comment]
@johnsaigle thanks, I missed that while refactoring the select for checking the multiple setting. I've also fixed the warning.

**Pull request merged**